lib/service.dart

```diff
@@ -192,6 +232,243 @@ class IsolateManager {
 
     return isolates.isEmpty ? null : isolates.first;
   }
+
+  void _initFlutterIsolate(List<IsolateRef> isolates) async {
+    for (IsolateRef ref in isolates) {
+      // Populate flutter isolate info.
+      if (_flutterIsolate == null) {
+        final Isolate isolate = await _service.getIsolate(ref.id);
+        if (isolate.extensionRPCs != null) {
+          for (String extensionName in isolate.extensionRPCs) {
+            if (extensionName.startsWith('ext.flutter.')) {
```

Comment:
logic for checking if extension names start with ext.flutter is repeated. Tweak it so it is only written once.

Reply:
Added a helper method `_isFlutterExtension` to check startsWith('ext.flutter.'). The two methods where it is called actually don't overlap, so we need both calls.